### PR TITLE
initial proposal

### PR DESCRIPTION
**Pull request by citricacid2**
## Describe your project in 1-3 sentences

I'm going to build a smart backpack that's loaded with all sorts of cool tech features! It'll include dynamic RGB, tons of expansion, and lots of power.

## Hack Club Slack handle

ayush

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
@citricacid2 this is sooo awesome! i'm wondering, are you following any guide of sorts for this? i'm also wondering, what made you decide to choose this specific microcontroller?

also, just to confirm, do you already have all of the tools you listed?

and, does your budget include shipping and taxes?

thank you!

File: citricacid2.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+I'm going to build a smart backpack that's loaded with all sorts of cool tech features!
+
+Feature breakdown:
+
+- I'm planning to include an integrated battery that can charge my laptop, phone, and other devices right inside the backpack
+- This battery will provide power for all of the essential circuits inside the backpack.
+- The outside will have accesible USB ports, and a detachable MagSafe charger.
+- The arm straps will be lined with independently addressable RGB LEDs that both provide aesthetic and practical benefits.
+- The back of the bag will have a large RGB display that can be used for aesthetic purposes or for safety, such as break lights, turn signals, etc.
+- The zippers will have conductive . The LEDs can be used to indicate this.
```
[the remark this answers is not in the document]
Comment (citricacid2) [reply citricacid2.md:21]: oops yeah, let me fix that

**Comment by citricacid2** [comment]
> this is sooo awesome! 

Thank you so much!

> i'm wondering, are you following any guide of sorts for this?

Not really, i've been looking at kickstarters for other "smart backpacks" and taking their features for inspiration!

> i'm also wondering, what made you decide to choose this specific microcontroller?

I noticed that it had an IMU and an incredibly small footprint. I didn't want to make any promises, but theoretically I could use the IMU to detect rapid breaking and activate rear LEDs as an automatic break light, which would be really useful while biking on city streets.

> does your budget include shipping and taxes?

oops, i forgot taxes. I think everything has free shipping however. I will make the changes soon, thank you!

------
Also, I just wanted to check, if I need to reallocate my budget later down the line, would I be free to do so? (if I clearly document what changes I made and why)

**Comment by bellesea** [comment]
> > this is sooo awesome!
> 
> Thank you so much!
> 
> > i'm wondering, are you following any guide of sorts for this?
> 
> Not really, i've been looking at kickstarters for other "smart backpacks" and taking their features for inspiration!
> 
> > i'm also wondering, what made you decide to choose this specific microcontroller?
> 
> I noticed that it had an IMU and an incredibly small footprint. I didn't want to make any promises, but theoretically I could use the IMU to detect rapid breaking and activate rear LEDs as an automatic break light, which would be really useful while biking on city streets.
> 
> > does your budget include shipping and taxes?
> 
> oops, i forgot taxes. I think everything has free shipping however. I will make the changes soon, thank you!
> 
> Also, I just wanted to check, if I need to reallocate my budget later down the line, would I be free to do so? (if I clearly document what changes I made and why)

@citricacid2 sounds good! please request a review again when you're ready for us to take a look hehe :) and yes 100%, you'd just chat w/ the hcb person you'd be in contact with then reflect the changes within your file! thanks :)

**Comment by citricacid2** [comment]
Ok, just added tax and fixed the thread problem, what now?

**Review by bellesea — APPROVED** [review]
@citricacid2 this is super cool and can't wait to see you work on this!!

a fabulous member of @hackclub/bank-ops will be reaching out within the next few days with next steps :) thanks for your patience!

**Review by MelSmith104 — COMMENTED** [review]
ahhh @citricacid2 this project sounds so amazing!!! i love love love the idea of building a smart backpack! im dm'ing you on slack right now to chat through next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**